README.md

```diff
@@ -101,3 +91,19 @@ repo.getIssues(false).ifPresent(issueData -> issueData.forEach(issue -> {
         System.out.println(comment.user.username + ": " + comment.body));
 }));
 ```
+
+### Further data processing
+
+The data extracted by this tool can be further processed, for example using the `run-issues.py` skript from the tool [`codeface-extraction`](https://github.com/se-sic/codeface-extraction). This organises and unifies the issue data into a single .list file. It also allows for synchronisation with data from other data extraction tools, such as `codeface`.
+
+### `Referenced` events
+
+`Referenced` events are events generated in an issue if a commit references that issue in its commit message. The intended behavior is that the event is present in the issue's event data, and the commit is again present in the related commits of the issue. This does not work if it is not possible to fetch that commit. In this case, the event still exists, but it contains a link to a commit that the api cannot resolve, meaning that no data about the commit can be accessed.
+Known causes of this include:
```

Comment:
I guess we spell all events with lower-case characters only, don't we? That is, even if we are at the beginning of a sentence / section here, we should spell code items exactly the way they appear in the code.